liveblog/liveblog
# LBSD-2523 Fixes: AMP theme live comments are disappearing in admin console

The problem was the missing `client_blog` parameter in AMP post comments. 
To test: 
- create a blog using amp theme
- enable commenting 
- submit a comment from the embed and see it working properly
